examples/synthetic/4_synthetic_locate.py

```diff
@@ -0,0 +1,76 @@
+"""
+This script runs the locate stage for the synthetic example described in the tutorial
+in the online documentation. 
+
+:copyright:
+    2020–2024, QuakeMigrate developers.
+:license:
+    GNU General Public License, Version 3
+    (https://www.gnu.org/licenses/gpl-3.0.html)
+
+"""
+
+from quakemigrate import QuakeScan
+from quakemigrate.io import Archive, read_lut, read_stations
+from quakemigrate.signal.onsets import STALTAOnset
+from quakemigrate.signal.pickers import GaussianPicker
+
+
+# --- i/o paths ---
+station_file = "./inputs/synthetic_stations.sta"
+data_in = "./inputs/mSEED"
+lut_out = "./outputs/lut/example.LUT"
+run_path = "./outputs/runs"
+run_name = "example_run"
+
+# --- Set time period over which to run locate ---
+starttime = "2021-02-18T12:03:50.0"
+endtime = "2021-02-18T12:06:10.0"
+
+# --- Read in station file ---
+stations = read_stations(station_file)
+
+# --- Create new Archive and set path structure ---
+archive = Archive(
+    archive_path=data_in, stations=stations, archive_format="YEAR/JD/STATION"
+)
+
+# --- Load the LUT ---
+lut = read_lut(lut_file=lut_out)
+
+# --- Create new Onset ---
+onset = STALTAOnset(position="centred", sampling_rate=100)
```

Comment:
Deliberate to use 100 Hz for this, and 50 Hz in detect?

Reply:
Will check.

Reply:
Yeah, I think my logic was 'decimate in time for detect', but if it throws extra confusion in, we can just put both to 100 Hz. Was saving a second or something per run.

Reply:
(not even—0.3 seconds!)